doc/contributing/releases.md

```diff
@@ -1075,20 +1075,18 @@ This script will use the promoted builds and changelog to generate the post. Run
 ### 19. Announce
 
 The nodejs.org website will automatically rebuild and include the new version.
-To announce the build on Twitter through the official @nodejs account, email
-<pr@nodejs.org> with a message such as:
+To announce the build on social media, please ping the @nodejs-social-team
+on offical slack channel.
+
+Node.js is also available on Bluesky and a release announcement can be
+reposted using [nodejs/bluesky](https://github.com/nodejs/bluesky) repository.
+
+The post content should be similar to:
```

Comment:
I think _should_ is a bit of a strong word here, the releaser def has ownership over the messaging in their post, what about:

```suggestion
The post content can be as simple as:
```